drivers/overlay/overlay.go

```diff
@@ -1023,6 +1024,11 @@ func (d *Driver) get(id string, disableShifting bool, options graphdriver.MountO
 	} else if d.options.mountOptions != "" {
 		opts = fmt.Sprintf("%s,%s", d.options.mountOptions, opts)
 	}
+
+	if d.options.mountProgram == "" && unshare.IsRootless() {
+		opts = fmt.Sprintf("%s,userxattr", opts)
```

Comment:
Shouldn't this be set only on kernel >= 5.11?

Ubuntu and Debian have been supporting rootless overlayfs and do not want userxattr opt here (for kernel < 5.11).

Reply:
we don't support the Ubuntu and Debian custom patch for rootless overlay.  The main reason is that on old kernels it is not allowed to create whiteout files without an overlay mount and that would complicate how image pulls are performed

Reply:
We also don't want kernel version checks, since these features can/will be back ported to RHEL8 and Centos Stream.